ehr/resources/schemas/ehr_lookups.xml

```diff
@@ -2754,6 +2757,9 @@
                     <fkColumnName>entityid</fkColumnName>
                 </fk>
             </column>
+            <column columnName="datedisabled">
```

Comment:
Looks like a bad merge. This column doesn't exist on this table.

Reply:
bad merge indeed! removed